buildpacks/builder.go

```diff
@@ -21,6 +20,12 @@ import (
 	"github.com/buildpacks/pack/pkg/logging"
 )
 
+// DefaultBuilderImages for Pack builders indexed by Runtime Language
+var DefaultBuilderImages = map[string]string{
+	"node": "gcr.io/paketo-buildpacks/builder:base",
+	"go":   "gcr.io/paketo-buildpacks/builder:base",
```

Comment:
Will this eventually have a key for all runtimes supported by paketo?

Reply:
Yes, and more specifically for all runtimes we support.  For users providing completely new runtimes/templates that we are not aware of and map here, they will need to set that value in their custom language pack's `manifest.yaml`